### ISSUE TITLE
Brain Dump of Portfolio Inspirations

### ISSUE THREAD

**Issue by carolynshin** [chingu-voyage4/Toucans-Team-10#4]
@kmicks and @ryantelli - how about we post pictures and links of portfolios we like as well as features (like @kmicks did [here](https://chingu-voyage4.slack.com/archives/C985YR72L/p1519066290000085)). After we do this, we can all vote and agree on what we want our portfolio to look like, what features we want, and go from there!

**Comment by carolynshin** [comment]
@kmick posted:

### http://hirestephanie.today/

Some reasons that I liked this as a project and a model:
1. It's a Single Page Website, which seems to be rather in vogue these days. 
2. It might be something we actually use to display our projects at some point.  Personally, I have been wanting to develop a personal portfolio kind of site for myself, but simply haven't gotten around to it. 
3.  It uses some nice CSS transitions that scroll to the page section when you click a link.  Those might be fun to learn and play with. 
4. There is a neat little effect with the navigation bar (it collapses when you scroll).  I'm not exactly sure what she's using for that, but i'd be interested in finding out.  
5. I know this is a matter of personal taste, but I like how she has her portfolio items laid out and I also like the hover effect she uses.  
6. Lastly, it looks like she is using Bootstrap on this site...I have been looking for a project to implement the new CSS Grid layout model, which I think we could definitely do instead of using Bootstrap on a page like this.  (or maybe flexbox is more suitable? I am honestly not 100% clear on when one is best to use over the other...but I would like to practice with either).

**Comment by carolynshin** [comment]
@kmicks ⬆️ , I absolutely love Stephanie's web page.
❤️  single page website
❤️  CSS animations (as you scroll down)
❤️  simplicity
❤️  nav bar links to sections of the page

**Comment by carolynshin** [comment]
### http://www.denisechandler.com/

❤️  water droplet animation from the Octopus legs 😄

**Comment by carolynshin** [comment]
### http://www.kathrynmcclintock.com/

❤️  Vertical picture looks interesting 

![screen shot 2018-02-19 at 5 43 53 pm](https://user-images.githubusercontent.com/6924712/36403881-7c1d0444-159c-11e8-8809-7a73a0b35cc8.png)

But I also like full width and height pictures as well like below

### https://imdev.in/

![screen shot 2018-02-19 at 5 44 48 pm](https://user-images.githubusercontent.com/6924712/36403902-98c916f0-159c-11e8-94e3-1db2f3330df0.png)

👎  I'm not really into the hamburger nav bar:

![screen shot 2018-02-19 at 5 45 19 pm](https://user-images.githubusercontent.com/6924712/36403914-adc58db8-159c-11e8-9b2c-0702bc20aa34.png)

**Comment by kmicks** [comment]
The [](http://www.denisechandler.com/) is very cool.  
Things I like:

1. Great octopus animation!  
2. I like how she has the octopus overlapping her 'about' image.  Very creative.
3. I like the grid layout and hover effect for the fonts she has for sale.  
4. I like how the icons are contained in a circle with a negative margin, causing them to overlay on top of the section above.

One concern I have about this site is how applicable it would be to use for our own portfolio sites.  It is so unique, that we would have to almost completely change it in order to use the layout to inspire our own sites.  That said, it is so cool, that I would be open to trying it if that is the consensus.  

[](http://www.kathrynmcclintock.com/)
My gut opinion is that I am personally just not a fan of this one.  :(  I'm also not sure there is enough to it to make it a challenging project between 3 people. 

[](https://imdev.in/)
I like the grid layout he has for his portfolio and the hover effect for each item.  I agree with you on the hamburger button @carolynshin  One concern is that we probably want to be coding the site from scracth, while this one is using WordPress & a theme.  Of course, we could replicate it with our own code, but how would we handle the blog section?  (I don't know how to do something like that without using a cms like WordPress, myself.

Okay...that's my morning thoughts.  I'm going to have a look for some additional inspiration ;)

**Comment by carolynshin** [comment]
Agreed with everything you said! Just wanted to clarify that I don't mean to replicate any of these layouts exactly. I just wanted to call out specific pieces/designs I liked from the different layouts. So we can come up with a list of features/designs we all mutually want to add to our project.

**Comment by kmicks** [comment]
Another site I came across this morning.  I'm actually not crazy about the site itself, but I think the typing effect at the top is pretty cool and I think would only require a small bit of javasript to implement.  Maybe something to consider adding to our project.
